api/v1/models/Model.js

```diff
@@ -29,7 +29,9 @@ Model.transaction = function (callback) {
  */
 Model.findById = function (id) {
```

Comment:
This function is not properly written. It will be fixed in `feature/storage-integration`